sprocs/check_join_group.sql

```diff
@@ -0,0 +1,70 @@
+CREATE OR REPLACE FUNCTION
+    check_join_group (
+        arg_assessment_id bigint,
+        arg_user_id bigint,
+        arg_authn_user bigint,
+        arg_group_name text,
+        arg_join_code text
+    ) RETURNS void
+AS $$
+DECLARE
+    arg_group_id bigint;
+    arg_cur_size bigint;
+    arg_max_size bigint;
+BEGIN
+    -- find group id
+    SELECT 
+        g.id
+    INTO 
+        arg_group_id
+    FROM 
+        groups AS g
+        JOIN group_configs AS gc ON g.group_config_id = gc.id
+    WHERE 
+        g.name = check_join_group.arg_group_name 
+        AND g.join_code = check_join_group.arg_join_code
+        AND gc.assessment_id = check_join_group.arg_assessment_id
```

Comment:
If we are using unique argument names that start with `arg_` then we don't need to add the `check_join_group.` at the start.